_security/access-control/document-level-security.md

```diff
@@ -10,30 +10,33 @@ redirect_from:
 
 # Document-level security (DLS)
 
-Document-level security lets you restrict a role to a subset of documents in an index. The easiest way to get started with document- and field-level security is to open OpenSearch Dashboards and choose **Security**. Then choose **Roles**, create a new role, and review the **Index permissions** section.
-
-![Document- and field-level security screen in OpenSearch Dashboards]({{site.url}}{{site.baseurl}}/images/security-dls.png)
-
-
-## Simple roles
-
-Document-level security uses the OpenSearch query DSL to define which documents a role grants access to. In OpenSearch Dashboards, choose an index pattern and provide a query in the **Document level security** section:
-
-```json
-{
-  "bool": {
-    "must": {
-      "match": {
-        "genres": "Comedy"
-      }
-    }
-  }
-}
-```
-
-This query specifies that for the role to have access to a document, its `genres` field must include `Comedy`.
-
-A typical request to the `_search` API includes `{ "query": { ... } }` around the query, but in this case, you only need to specify the query itself.
+Document-level security lets you restrict a role to a subset of documents in an index.
+For more information about users and roles in OpenSearch[, see the documentation.](https://opensearch.org/docs/latest/security/access-control/users-roles/#create-roles)
+
+The easiest way to get started with document- and field-level security is:
+1. Open OpenSearch Dashboards 
+2. Choose **Security**
+3. Choose **Roles** 
+4. Create a new role
+5. Review the **Index permissions** section
```

Comment:
What are users looking for as they review the **Index permissions** section?

Reply:
Inside the 'index permissions' area/heading, is the DSL multiline text box, so it was just a step to get to the DSL area.